clients/package.json

```
@@ -0,0 +1,33 @@
+{
+  "workspaces": [
+    "admin-ui/",
+    "privacy-center/",
+    "fides-consent/"
+  ],
+  "scripts": {
+    "build": "turbo run build",
+    "build-admin-ui": "turbo run build --filter=admin-ui",
+    "build-pc": "turbo run build --filter=privacy-center",
```

```suggestion
    "build-privacy-center": "turbo run build --filter=privacy-center",
```

Sorry but I think it'll be important to be clear about this 😄 

Also, shouldn't you have some options for building the fides-consent package?

---

I can add some. Right now, because `fides-consent` is a dependency of privacy center, it gets automatically built as part of `build-privacy-center`. Right now, you can `cd` into `fides-consent` and run `turbo run build` and that will work. The npm stuff in base `package.json` is more for convenience, particularly when needing to run a particular command across all workspaces